examples/src/chains/sql_db.ts

```diff
@@ -17,11 +19,88 @@ const db = await SqlDatabase.fromDataSourceParams({
   appDataSource: datasource,
 });
 
-const chain = new SqlDatabaseChain({
-  llm: new OpenAI({ temperature: 0 }),
-  database: db,
+const llm = new ChatOpenAI();
+
+/**
+ * Create the first prompt template used for getting the SQL query.
+ */
+const prompt =
```

Comment:
Would show how to load our default prompts for MySQL, Postgres, etc. in this case

Reply:
Updated!